testsuite/end2end/server_file_example.yml

```diff
@@ -0,0 +1,57 @@
+# WBEM server definition file for pywbem end2end tests
+#
+# This file has the following format:
+#
+# servers:                       # Dict of WBEM servers.
+#   NICKNAME1:                   # Nickname of the WBEM server.
+#     description: DESC          # Short description of the WBEM server.
+#                                # Optional, default: empty.
+#     url: URL                   # URL of the WBEM server; for details see url
+#                                # parameter of pywbem.WBEMConnections().
+#                                # Mandatory.
+#     user: USER                 # User for logging on to the WBEM server.
+#                                # Mandatory.
+#     password: PASSWORD         # Password of that user.
+#                                # Mandatory.
+#     cert_file: CERT_FILE       # Path name of file containing X.509 client
+#                                # certificate to be presented to server, or
+#                                # null for not presenting a client certificate
+#                                # to the server.
+#                                # Optional, default: null.
+#     key_file: KEY_FILE         # Path name of file containing X.509 private
+#                                # key of the client certificate, or null for
+#                                # not presenting a client certificate to the
+#                                # server.
+#                                # Optional, default: null.
+#     ca_certs: CA_CERTS         # Path name of CA certificate file or path
+#                                # name of directory containing CA certificate
+#                                # files to be used for verifying the returned
+#                                # server certificate, or null for using the
+#                                # pywbem default locations.
+#                                # Optional, default: null.
+#     no_verification: BOOL      # Boolean controlling whether the returned
+#                                # server certificate is to be verified.
+#                                # Optional, default: true.
+#
+# server_groups:                 # Dict of WBEM server groups (optional).
+#   NICKNAME42:                  # Nickname of WBEM server group.
+#     - NICKNAME1                # Nicknames of servers in that group.
+#     - NICKNAME2
+
+# Note: This is an example file that should be copied to server_file.yml and
+# modified as needed.
+
+servers:
+  srv1:
+    description: My WBEM server
+    url: https://srv1.xxx.com:5989
+    user: USER
+    password: PASSWORD
+    cert_file: ~/.ssh/cert_file
+    key_file: ~/.ssh/key_file
+    ca_certs: ~/.ssh/ca_certs/
+    no_verification: false
+
+server_groups:
```

Comment:
Should the example file show the default entry, probably as a group ???

Reply:
good point

Reply:
DONE - added default group